### PR TITLE
--list, --list-all and --single-column options

### PR DESCRIPTION
**Pull request by igorshubovych**
New and updated options:
- `-l` or `--list` - print all commands for default platform
- `-a`, `--list-all` - print all commands for all platforms
- `-1` or `--single-column` - print commands in a single column, has to be used together with `--list` or `--list-all`

### CODE REVIEW

**Comment by igorshubovych** [comment]
It fixes #50.

**Comment by igorshubovych** [comment]
Sorry for useless commits, i tried really hard (#65).

File: lib/cache.js [hunk 1]
```diff
@@ -62,9 +72,14 @@ function tryLoad(pages, index, done) {
 }
 
 function isPage(file) {
-  return file.match(/pages\/.*\.md$/);
+  return file.match(/.*\.md$/);
 }
 
+function onPlatform(os) {
+  return function(file) {
+    return _.startsWith(file, 'common') || _.startsWith(file, os);
```
Comment (rubenvereecken) [inline lib/cache.js]: Tssk including the whole of lodash for this :P? 

`file.match(/^common/)` for the first bit, second you can build from string.

I really don't care though, lodash is great.

**Comment by rubenvereecken** [comment]
Great stuff :). I managed to (kinda) duplicate the `-1` using `tldrl -l | tr ',' '\n'` because I was curious if it was needed, but great PR.

**Comment by igorshubovych** [comment]
Yes, I know that `tr` can help here. But it is so convenient.

**Comment by rubenvereecken** [comment]
I merged it because it was so pretty.

File: lib/cache.js (hunk 1 above)
Comment (igorshubovych) [inline lib/cache.js]: Lodash is used in different places already.

**Pull request merged**